### DISCUSSION TITLE
How to export the knowledge map to facilitate subsequent retrieval and call by other computer

### DISCUSSION THREAD

**Discussion by yzyz-77** [OpenSPG/openspg#417] [Q&A]
I hope ask how to export the knowledge map secondary generated after analyzing it on Linux cloud server according to KAG tutorial, and then put it on windows notebook or other cloud servers to facilitate subsequent retrieval and call by other computer.

**Comment by yzyz-77** [comment]
KAG tutorial used is https://openspg.yuque.com/ndx6g9/0.5/otced6rb3z4vtq2y

**Comment by caszkgui** [comment]
> I hope ask how to export the knowledge map secondary generated after analyzing it on Linux cloud server according to KAG tutorial, and then put it on windows notebook or other cloud servers to facilitate subsequent retrieval and call by other computer.

Currently, you can browse extracted kg data by visiting neo4j: [neo4j brower](https://openspg.yuque.com/ndx6g9/0.5/nbb1bn3wegwue6yo#Z3EP9), where ip is the host_addr of neo4j server

**Reply by yzyz-77** [reply]
But the cloud server I rented can't open his browser. What command or code can I use to extract his data?Thanks
